### PR TITLE
Update Android to use flutter_inappwebview

### PR DESCRIPTION
**Pull request by atavism**

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
     <uses-permission android:name="com.android.vending.BILLING" />
     <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES" />
     <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
+    <uses-permission android:name="android.permission.READ_PRIVILEGED_PHONE_STATE" />
```
Comment (atavism) [inline android/app/src/main/AndroidManifest.xml:24]: Fix for the following error:

```
  /Users/paul/.pub-cache/hosted/pub.dev/device_info_plus-10.1.2/android/src/main/kotlin/dev/fluttercommunity/plus/device_info/MethodCallHandlerImpl.kt:70: Error: Missing permissions required by Build.getSerial: android.permission.READ_PRIVILEGED_PHONE_STATE [MissingPermission]
                      Build.getSerial()
```

**Comment by atavism** [comment]
I don't think we need WebViewActivity anymore with these changes, but the survey stuff is connected to it so I left it for now.

**Review by jigar-f — APPROVED** [review]
LGTM

**Comment by jigar-f** [comment]
@atavism  The only thing is WebViewActivity was using a proxy setting and now in flutter, we are not using it.

**Comment by atavism** [comment]
> @atavism The only thing is WebViewActivity was using a proxy setting and now in flutter, we are not using it.

Oh, great catch! Let me fix that.

**Comment by atavism** [comment]
Ok, I'm using ProxyController setProxyOverride to configure the proxy the webview uses now too

**Pull request merged**